delivery_package_sscc/readme/USAGE.rst

```diff
@@ -0,0 +1,18 @@
+To use this module, you need to:
+
+* Identify your SSCC range. For example: `123456789012xxxxx`.
+* Go to the form view of the `ir.sequence` record for your packages.
```

Comment:
I will make this bullet point less technical, maybe by adding the menu path to follow?
```suggestion
* Go to the form view of the sequence record for your packages.
```